.eslintrc

```diff
@@ -4,7 +4,21 @@
     "jest/no-conditional-expect": "off",
     "radar/no-identical-functions": "off",
     "unicorn/no-array-callback-reference": "off",
-    "unicorn/no-array-for-each": "off"
+    "unicorn/no-array-for-each": "off",
+    "unicorn/prefer-object-from-entries": "off",
```

Comment:
We want to change reduce to fromEntries. Let's not add this rule, rather change the code.

Reply:
From what I recall, @eaviles mentioned to just disable this rule for this iteration for now. I think the "actual fixes" will come with core-commons.